Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -128,15 +150,22 @@ class SentryUserFeedbackForm: UIViewController {
         
         guard missing.isEmpty else {
             let list = missing.count == 1 ? missing[0] : missing[0 ..< missing.count - 1].joined(separator: ", ") + " and " + missing[missing.count - 1]
-            let alert = UIAlertController(title: "Error", message: "You must provide all required information. Please check the following field\(missing.count > 1 ? "s" : ""): \(list).", preferredStyle: .alert)
+            return "You must provide all required information. Please check the following field\(missing.count > 1 ? "s" : ""): \(list)."
```

Comment:
`l`: should we support localizations?

Reply:
If I'm not mistaken, the plan is to add this later. @armcknight can confirm.

Reply:
Yes, it would be a future thing but i'm not even sure where that is on the roadmap. I'm not familiar with any kind of localization for any of our products.

Reply:
> I'm not familiar with any kind of localization for any of our products.

Me neither, because we mostly build for devs and they usually speak English 😅. We'll figure that out later.

Reply:
@philprime, I know, that's why we most likely will need localization.

Reply:
Correct @philprime, we can look to the existing offering for user feedback in the javascript sdk: https://github.com/getsentry/sentry-javascript/blob/develop/packages/feedback/src/constants/index.ts

I don't think there are other places to look for translated user-facing strings in that implementation. I'll ask in the user feedback project's channel what the thinking is for i18n.